google/ggrc-core
# GGRC-5829 Re-name 'Text of Requirement' attribute into 'Description'

# Issue description
Re-name 'Text of Requirement' attribute into 'Description' for Requirement objects

# Steps to test the changes

1. Check export csv for Requirement object, "Text of Requirement" field will be replaced with "Description"
2. Check import csv for Requirement object, "Description" field will be processed correct
3. Run "re-index" task and check global search by "description" field for Requirement
4. Run integration tests
 
# Solution description

Replace "Text of Requirement" in _aliases dictionary for Requirement model
Update csv files, replace "text of requirement" with "description"

# Sanity checklist


# PR Review checklist
